addons/website_sale_slides/views/website_sale_templates.xml

```diff
@@ -21,7 +21,7 @@
                     <span t-if="course.image_1920" t-field="course.image_1920" t-options="{'widget': 'image', 'class': 'my-2'}"/>
                     <img t-else="" class="img img-fluid my-2" src="/website_slides/static/src/img/channel-training-default.jpg"/>
                 </div>
-                <t t-set="invitation_link" t-value="course_memberships[course].invitation_link"/>
+                <t t-if="course in course_memberships" t-set="invitation_link" t-value="course_memberships[course].invitation_link"/>
```

Comment:
I don't like that this variable can be unset, what do you think about:
```suggestion
                <t t-set="invitation_link" t-value="course_memberships[course].invitation_link if course in course_memberships else '#'"/>
```

Reply:
@Julien00859 yes, might be better to always have it just in case, it makes sense. I only made one small modification to the suggestion, is it possible to set it to an empty string or something similar instead of '#'.  Why this? I think that It's better that when there's no link available the button stays disabled, if we just set '#' the user will still be able to click the button but it will do nothing, which can be confusing. Thanks for the review!

Reply:
@Ephoniz an empty string is very fine!